WareDrop_Backend/waredrop_backend/src/app.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
+import { AppService } from './app.service';
+
+@Controller()
+export class AppController {
+  constructor(private readonly appService: AppService) {}
+
+  @Get()

```

Comment:
nem kell a hello world endpoint, törölheted. Másik fájlokban is van erre vonatkozó kód, az is legyen kitörölve vele együtt